pgbouncer/kubedb_client_builder.go

```diff
@@ -164,6 +165,46 @@ func (o *KubeDBClientBuilder) getBackendAuth() (string, string, error) {
 	return string(user), string(pass), nil
 }
 
+func (o *KubeDBClientBuilder) getTLSConfig() (*certholder.Paths, error) {
+	secretName := ""
+	secretNamespace := ""
+
+	if o.databaseName == "pgbouncer" {
+		secretName = o.pgbouncer.GetCertSecretName(api.PgBouncerClientCert)
+		secretNamespace = o.pgbouncer.Namespace
+	} else {
+		// is database name isn't "pgbouncer" then the database is from postgres and need to find the CA secret from appbinding
```

Comment:
What if PgBouncer and Postgres is not same CA?

Reply:
solved